libreyolo/models/sam/base.py

```
@@ -0,0 +1,604 @@
+        raise NotImplementedError(
+            f"Training is out of scope for {type(self).__name__} in LibreYOLO. "
+            "Fine-tune the model upstream and load the resulting weights."
+        )
+
+    def val(self, *args, **kwargs):
+        raise NotImplementedError(
+            f"Dataset validation is not supported for {type(self).__name__}: "
+            "promptable masks have no fixed class set to score against."
+        )
+
+    def export(self, format: str = "onnx", **kwargs) -> str:
```

Ruff (0.15.15)

[error] Function argument `format` is shadowing a Python builtin

(A002)